src/main/java/org/opensearch/knn/index/KNNSettings.java

```diff
@@ -97,6 +98,7 @@ public class KNNSettings {
      */
     public static final boolean KNN_DEFAULT_FAISS_AVX2_DISABLED_VALUE = false;
     public static final String INDEX_KNN_DEFAULT_SPACE_TYPE = "l2";
+    public static final boolean INDEX_KNN_DEFAULT_BUILD_GRAPH = true;
```

Comment:
rename this as per the new setting name I suggested on top.

Reply:
ACK